packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -231,7 +244,7 @@ const project = new codebuild.Project(this, 'MyProject', {
     }),
   ],
   secondaryArtifacts: [
-    new codebuild.S3BucketBuildArtifacts({
+    new codebuild.S3Artifacts({
```

Comment:
Do we want union-like for `codebuild.Artifacts` as well (e.g. `codebuild.Artifacts.s3(...)`)

Reply:
We already have it.